<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>561</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="Input">
       <property name="title">
        <string>original face</string>
       </property>
       <widget class="QLabel" name="original_face1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="original_face2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="face_landmark">
       <property name="title">
        <string>face landmark detection</string>
       </property>
       <widget class="QLabel" name="landmark1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="landmark2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="face_alignment">
       <property name="title">
        <string>face alignment</string>
       </property>
       <widget class="QLabel" name="aligned_face1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="aligned_face2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="CNN">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;CNN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="originalFaceShow">
       <property name="text">
        <string>show original face </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="faceLandmarkDetection">
       <property name="text">
        <string>face landmark detection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="faceAlignment">
       <property name="text">
        <string>face alignment</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>151</width>
      <height>216</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="prob_label">
       <property name="text">
        <string>prob value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="prob_value"/>
     </item>
     <item>
      <widget class="QLabel" name="result_label">
       <property name="text">
        <string>verification result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="Verification_Result"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="START">
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="QUIT">
       <property name="text">
        <string>QUIT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="LoadingFace1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>loading face 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadingFace2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>loading face 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
